AGENTS.md

```
@@ -0,0 +1,300 @@
+# AGENTS.md
+
+Purpose
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)